packages/replay/src/util/debounce.ts

```diff
@@ -57,8 +57,13 @@ export function debounce(func: CallbackFunction, wait: number, options?: Debounc
     }
     timerId = setTimeout(invokeFunc, wait);
 
-    if (maxWait && maxTimerId === undefined && maxWait !== wait) {
-      maxTimerId = setTimeout(invokeFunc, maxWait);
+    if (maxWait && maxTimerId === undefined) {
```

Comment:
Actually, `invokeFunc` already calls `clearTimeout`, so we can streamlime this slightly - IMHO we only need to remove the `maxWait !== wait` part, which was an incorrect "streamlining" that we added.

I'll adjust this and merge this and will actually release this fix today as well!